infra/base-images/base-runner/run_fuzzer

```diff
@@ -106,9 +106,10 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
   export AFL_NO_AFFINITY=1
   export AFL_FAST_CAL=1
   export AFL_MAP_SIZE=4194304
+  test -e $OUT/afl_cmplog.txt && AFL_FUZZER_ARGS="$AFL_FUZZER_ARGS -l 2 -c $OUT/$FUZZER"
```

Comment:
can you please add a comment on what is -l 2 or what does this argument addition do.

Reply:
Not sure how much comment you want - I might have gone overboard ;)

Reply:
This is great!